Orbitersdk/samples/ProjectApollo/src_rtccmfd/ApolloRTCCMFD.cpp

```diff
@@ -1560,6 +1560,39 @@ void ApolloRTCCMFD::menuSaveDODREFSMMAT()
 	GeneralMEDRequest("G11,CSM,DOM;");
 }
 
+void ApolloRTCCMFD::menuSaveRTEREFSMMAT()
+{
+	bool SaveRTEREFSMMATInput(void *id, char *str, void *data);
+	oapiOpenInputBox("Save RTE REFSMMAT. P for primary or M for manual column.", SaveRTEREFSMMATInput, 0, 20, (void*)this);
+}
+
+bool SaveRTEREFSMMATInput(void *id, char *str, void *data)
+{
+	if (strlen(str) < 3)
+	{
+		char code[4];
+		if (str[0] == 'P')
+		{
+			sprintf(code, "REP");
```

Comment:
You can avoid the call to sprintf (expensive) and assign code directly if you turn it into a pointer.
Feel free to cherry-pick: https://github.com/ThymoNL/NASSP/commit/7e559049f97376c2f39743f690e41f8ec2a185ab